docs/nidigital/class.rst

```diff
@@ -1564,7 +1564,12 @@ fetch_capture_waveform
 
     .. py:method:: fetch_capture_waveform(site_list, waveform_name, samples_to_read, timeout)
 
-            TBD
+            Returns a list of named tuples (Waveform) that <FILL IN THE BLANK HERE>
+
+            Fields in Waveform:
+
+            - **site** (int)
+            - **data** (array.array of int)
```

Comment:
* Is `array.array of int` the right type for the data of digital waveforms?
* When I think of `Waveform` I think of an analog signal by default. Should we call this DigitalWaveform or is it enough that it is an `nidigital.Waveform`?